cli/popper/utils.py

```diff
@@ -118,7 +118,7 @@ def fail(msg):
 
 
 def warn(msg):
-    click.secho('WARNING: ' + msg, fg='orange', bold=True)
+    click.secho('WARNING: ' + msg, fg='magenta', bold=True)
```

Comment:
@ankan17 Good spot! Maybe change it to "yellow" instead of "magenta" ?
I did some research on this and found out that the usual color code for warnings is something in between yellow and orange. Also, according to the [Click Documentation](http://click.pocoo.org/5/api/#click.secho), sometimes "yellow" can behave as an "orange" i.e. it will be consistent with our ideas both ways.